<launch>

<node pkg="car_ctr" name="car_ctr" type="car_ctr" output="screen">
    <param name="can_enable" value="false"/>
    <param name="remote_ip" value="192.168.1.101"/> 
</node>

<node pkg="car_ctr" name="car_charge" type="car_charge" output="screen">
    <param name="remote_ip" value="192.168.3.100"/> 
</node>

<!-- <node pkg="rqt_gui" type="rqt_gui" name="rqt_gui"/> -->

</launch>